lib/rubocop/config_loader.rb

```diff
@@ -174,10 +174,15 @@ def check_duplication(yaml_code, absolute_path)
         smart_path = PathUtil.smart_path(absolute_path)
         YAMLDuplicationChecker.check(yaml_code, absolute_path) do |key1, key2|
           value = key1.value
-          line1 = key1.start_line + 1
-          line2 = key2.start_line + 1
-          message = "#{smart_path}:#{line1}: " \
-                    "`#{value}` is concealed by line #{line2}"
+          message = if key1.respond_to? :start_line
```

Comment:
Please add some comment here about the need for the extra check, so we can delete it down the road.

Reply:
will do 👍